pkg/registries/topo.go

```diff
@@ -1,4 +1,4 @@
-package containerruntimeconfig
+package registries
```

Comment:
can this pkg be under `pkg/controller/container-runtime-config`?

Reply:
Sure; I moved it up here to make it easier to find / read for out-of-repo consumers (who don’t care about the M-C-controller much).

Ideally we would want to have it somewhere out of the 2 (3? 4?) known users into some shared infrastructure repo.

If it is going to stay in MCO, I really don’t care where.

Reply:
Moved.